addons/base_vat/models/res_partner.py

```diff
@@ -34,7 +34,7 @@
     'co': _('CO213123432-1 or CO213.123.432-1'),
     'cy': 'CY10259033P',
     'cz': 'CZ12345679',
-    'de': 'DE123456788',
+    'de': ('DE123456788 or 12/345/67890'),
```

Comment:
I'm sorry I missed it last time. 
It should be translatable (for the `or`), so you need to add a `_` here, and add it to the .pot, as we're in stable